### ISSUE TITLE
Is there any someone use this boilerplate for production ?

### ISSUE THREAD

**Issue by andrechristikan** [andrechristikan/ack-nestjs-boilerplate-mongoose#458]
Let me know about what your feedback while using this boilerplate

**Comment by dcapinator** [comment]
trying to reach u insta/linkedin , lets connect?

**Comment by 0xdecapitator** [comment]
i am trying to reach u out on linkedin and instagram , lets connect!!

**Comment by yonashalefom** [comment]
Yes we're using this for production project. This repo is really awesome!

**Comment by andrechristikan** [comment]
Lets connect !

I very rarely open LinkedIn.
You guys can add me on Instagram or my email `andrechristikan@gmail.com`

**Comment by thenna** [comment]
Yep - I have started using this in one of the client projects. As @Yon12853  told - This repo is really awesome! and learned a lot!!!!

**Comment by truongbo17** [comment]
I wrote a nestjs project inspired by this repository, which has been used on client and product projects for my company. It's really great despite some minor flaws.
